### ISSUE TITLE
Improve discoverability and the ways to join the weekly/fortnightly Google Hangout meetings.

### ISSUE THREAD

**Issue by klonos** [backdrop-ops/backdropcms.org#215]
I was so disappointed today! ...I made a decision last night to wake up early in the morning in order to finally join the weekly dev meeting (so far, I've been only watching the recording on YouTube). I tried to figure out how to do that (never used Google Hangouts before) and I swear it's hard. ...mind you I consider myself very computer-savvy and I've been around for 17+ years now.

So, I started with our [Resources and Tools](https://backdropcms.org/resources) page. The "Weekly Meetings: Google hangouts" section prompts you to join our [Google+](https://plus.google.com/+BackdropcmsOrg) group. So, I did that, but there was no obvious way to see the scheduled meetings or how you go about it. Is there a desktop app I need to download or is it all done on my browser? Is my default browser (firefox) compatible with the Hangouts platform or do I need Google Chrome?

I often hear you guys mention IRC in the meetings, so I thought I might give that a go for pointers. I downloaded mIRC (have not used it since the 1990s BTW) and joined the channel mentioned on our [Resources and Tools](https://backdropcms.org/resources) page. I asked there and was pointed to our Tweeter feed (I do not use it, so I was lucky that it doesn't need registration) that has a link to the Google Doc with the meeting agenda that is supposed to have a link that you can use in order to join. Did that, but the only link available was the one to watch the recorded video on YouTube. Perhaps I was too late and the meeting has ended. Fair enough, but boy what a hustle! I know now for the next time, but I was so excited by the idea of finally joining that the disappointment was huge :disappointed: 

Lets recap: website -> IRC -> Tweeter -> Google Docs -> no-go ...to many steps involved even if I finally got to it. So, lets fix this as much as we can and make it easy for people to join the meetings if they wish to:

- There is a "Meet Up" section in our [Ways to Contribute](https://backdropcms.org/contribute) page with links to the iCal feeds of the meetings. I think that these should be copied over to the [Resources and Tools](https://backdropcms.org/resources) page in the "Weekly Meetings: Google hangouts" section. I'll take care of that now that I have permissions to edit those pages.

- I have imported these on my calendar application, so I now know *when* the next meeting is held, but the question remains: *how* do I actually join? I am a Skype user, so I am used to the fact that I need to send an invite to a contact and they need to accept it before I can confer with them. How does it work with Google Hangouts? How is the meeting initiated and will I get some sort of ringing/call the way that Skype works?

- ...

**Comment by klonos** [comment]
> ...the only link available was the one to watch the recorded video on YouTube. Perhaps I was too late and the meeting has ended.

IRC excerpt:

```
<klonos> I just took a look at the Google doc linked from the twitter feed and it has only a link to watch the meeting on YouTube...
<klonos> I guess because the meeting mast have ended by now :(
<klonos> ...I'll file an issue in the backdropcms.org queue about this and perhaps also update the "Resources and Tools" page in https://backdropcms.org/resources to include some info about how to join.
<klonos> ...the "Weekly Meetings: Google hangouts" section in that page has a link to join through Google+. I did that, but I did not see any obvious way to join the meeting from there. Trust me, if it was hard for me to figure things (17+ years of experience in IT), I bet you it must be hard for newcomers.
<klonos> ...the "Ways to Contribute" page in https://backdropcms.org/contribute has a "Meet Up" section (at the very bottom of the page though) with links to iCal files. I think that these should be in the "Resources and Tools" page as well.
<jackalope> klonos: I think it's because the meeting is tomorrow, not today!
* albert1t0 (~Thunderbi@187.189.10.65) Quit (Remote host closed the connection)
<klonos> jacklope: http://www.timebie.com/timezone/pacificmelbourne.php
<klonos> ...it's **next day**. Silly me.
```

...no comments please :smile:

**Comment by docwilmot** [comment]
Quick Google search for "Static Hnagout Link" brings this: http://behindcompanies.com/2014/01/set-up-a-permanent-google-hangout-for-easy-collaboration/. We should do that, if it works, because as you say hunting the hangout is ridiculous. Only Just accidentally figured out how to join late last year when I downloaded Twitter app for an entirely different purpose.

**Comment by serundeputy** [comment]
@klonos the meeting is on Thursdays 3/31 tomorrow: https://docs.google.com/document/d/11dPo2w_6jjTt_WXjvpN6ePScafucACLugKOJyTOqz_I/edit?usp=sharing 

The easiest way to be informed of the latest and greatest link is: https://gitter.im/backdrop/backdrop-issues which is also tied to our issue queue just visit the gitter link in any browser.  Join us tomorrow.

**Comment by klonos** [comment]
Thanx you guys! @docwilmot Yes, lets do that if it works. Would be easier to have a permanent link in our Google Hangouts section. @serundeputy Will do. See you guys live tomorrow!

**Comment by docwilmot** [comment]
@klonos this would be one for @jenlampton to do; I dont have access to the Google+ site.

**Comment by jenlampton** [comment]
Unfortunately I don't think that will work, as that post says "Your Hangout link will expire on this day". Let me see if there is a way to set up a recurring event and "attach" a hangout to it. (I tried this before but was unsuccessful...  since Backdrop is a G+ "page" and not a person, the calendar doesn't work like it should. Maybe they have fixed that!

**Comment by jenlampton** [comment]
I tried scheduling a hangout and creating an event, neither one lets me set the hangout URL. Maybe it's not possible for "hangouts on air"?

**Comment by jenlampton** [comment]
if anyone else wants to battle with this G+ page and try to help figure this out, I'd be happy to add you :) the whole thing is a mystery to me!

**Comment by biolithic** [comment]
It can be done -- I remember John Albin in Taiwan, many others around the
globe and I in America attending weekly mobile D8 meetings, and that was
kind of a pain for everyone.  I have had (non-Drupal) clients in Australia,
we remember the meetings somehow.

I've never attended a scheduled Hangout.  "When it is time", somebody just
creates one and then we have the meeting.  Sorry for the lack of help on
that account.  I just pay attention to some type of real time social
communication around the scheduled time and then click the link in my
browser.

**Comment by biolithic** [comment]
"real-time communication" could be your pick of irc, twitter, g+, gitter,
any others?  Fbook, Myspace, Friendster?  Bueller?

**Comment by biolithic** [comment]
I guess the Backdrop community on Friendster is taking a break effective
June 14, 2015, so don't be listening to that :)

**Comment by klonos** [comment]
I just find it silly (especially at a time where we are looking to attract more people to b.org) to have them "jump through hoops" in order to find and join the meeting. That's all.

So to understand how this thing works better, @jenlampton do you initiate the hangout meeting a few minutes before the scheduled time and then get a URL for it that you twit through our official Twitter account. Is that how it goes?

If we cannot get a permanent link to a Hangout, and if twitting about the link is our only current option, then may I propose the following change to the workflow:

How about we make the meeting announcements a b.org node or something (so that we publish the link to the hangout on b.org **first**) and then have the [Twitter](https://backdropcms.org/project/twitter) module auto-twit that node on the Twitter account? That way

1. we do not disrupt the way of people currently discovering the URL
2. at the same time we have it published on the official website
3. since only publishing a node is all it takes to publish the URL in both places (the module will take care of auto-twitting it), there is no additional steps for the person that starts the hangout. They will simply publish a node on b.org instead of twitting.

What do you think?

**Comment by quicksketch** [comment]
> So to understand how this thing works better, @jenlampton do you initiate the hangout meeting a few minutes before the scheduled time and then get a URL for it that you twit through our official Twitter account. Is that how it goes?

Yep, currently that is the situation. We have not been able to figure out how to have a regularly scheduled hangout at the same URL. They are always unique and have to be set up at the time it happens.

> How about we make the meeting announcements a b.org node or something

This is (sort of) what we do already, though we could adjust. Right now we tweet the meeting agenda, which as you know does NOT contain the link (or should not) until just ~5 minutes before the meeting time. Then we (Jen) updates the agenda with the hangout URL. Right now the best way of finding the link is going to the agenda, waiting until meeting time, and then it will be updated as soon as it is created.

We could expand this to process to create nodes on bdcms.org as well. Right now we only update the Google doc as things are happening, including frequent updating during the meeting itself as discussion takes place.

**Comment by klonos** [comment]
Fair enough. I understand that there are technical limitations with the way Google Hangouts are created and that they cannot be scheduled and/or have a permanent URL. We'll simply have to live with it and adapt.

Is it ok if I go forward with the things I proposed in the issue summary?

**Comment by jenlampton** [comment]
I don't like the idea of having a node for each meeting on the website. It seems not only like too much for work for running the meetings, but also too much clutter on the site, as most people there won't be intending to join the meetings. 

How about adding a single block somewhere? Something that says "watch twitter for meeting info" with a link to the twitter feed?

**Comment by quicksketch** [comment]
> There is a "Meet Up" section in our Ways to Contribute page with links to the iCal feeds of the meetings. I think that these should be copied over to the Resources and Tools page in the "Weekly Meetings: Google hangouts" section. I'll take care of that now that I have permissions to edit those pages.

Yes that would be great. :smile:

**Comment by docwilmot** [comment]
I just went to my G+ acct and created a Hangouts on Air event with a future date (3 months from now) and got a link to share. Even if it turns out to be not a static link (I dont know, didnt explore that much) at the very least we can prepare the link for the next meeting a lot longer than 5 minutes before it seems.

**Comment by klonos** [comment]
@docwilmot maybe the key difference is what @jenlampton said previously:

> ...Backdrop is a G+ "page" and not a person, the calendar doesn't work like it should.

**Comment by BWPanda** [comment]
We don't use Google Hangouts, etc. anymore, so is this issue still relevant?
